blog/content/post/2021-10-28-rockchip-secure-boot.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Enabling Secure Boot on RockChip SoCs
+abstract: 'Secure Boot is essential for secure IoT infrastructures. It protects
+          IoT devices from being permanently infected and controlled by an
+          Attacker. In this post, I will tell a story about enabling Secure Boot
+          on RockChip SoC.
+          '
+cover: /covers/rockchip-logo.jpg
+author: artur.kowalski
+layout: post
+published: true
+date: 2021-10-28
+archives: "2021"
+
+tags:
+  - secure-boot
+  - firmware
+  - rockchip
+  - rk3288
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+RockChip is a Chinese company manufacturing low-cost ARM SoCs, yet with good
+performance. Another advantage is reasonably good support for mainline Linux
+with many drivers being already there. There are publicly available datasheets
```

Comment:
@pietrushnic Yes, I passed it to grammarly, but it didn't show many errors, sometimes in one place it was highlighting some error, but omitting the same error when it occured somewhere else.
I looked through the entire text and corrected it to avoid using these phrases. I hope everything is clear now.